### PR TITLE
Scheduler Cleanup for Issue #433

### PR DESCRIPTION
**Pull request by jtronge**
This should address most of issue #433. I'm also thinking about removing some of the modules that aren't really useful as is, such as the MARS component.

I would also like to try to do a refactor of the scheduler in a future PR. Scheduler improvements are documented in #333.

### CODE REVIEW

**Comment by pagrubel** [comment]
If this is ready for review please remove the WIP label

File: setup.cfg [hunk 1]
```diff
@@ -15,14 +15,14 @@ ignore = R0902,R0903,R0904,R0912,R0913,R0914,R0916,W0603,W1203,C0413
...
 max_line_length = 99
-good_names = i,j,k,v,t,m,s,n,ex,fp,id,tx,fn,Run,_
+good_names = i,j,k,v,t,m,s,n,x,y,ax,ex,fp,id,tx,fn,pi,Run,_
 
 [pylama:pycodestyle]
 max_line_length = 99
-ignore = E402
+ignore = E402,W503
 # E402: module level imports should be at the top of the file
+# W503: this frequently conflicts with the line length warning
```
Comment (pagrubel) [inline setup.cfg]: What is the warning "this"

File: src/beeflow/scheduler/train.py [hunk 2]
```diff
@@ -123,3 +122,12 @@
     actor.predict(vec)
     critic.predict(vec)
     mars.save_models(actor, critic, args.trained_model)
+
+
+if __name__ == '__main__':
+    main()
+# Ignore R0915: This requires a refactor related to issue #333.
```
Comment (pagrubel) [inline src/beeflow/scheduler/train.py]: Again say what you are ignoring

**Comment by pagrubel** [comment]
If we are going to ignore pylama explain what you are ignoring so someone reviewing the code knows what it is, otherwise all looks good

**Comment by jtronge** [comment]
Ok thanks. I think I've updated most of these. I hope to remove a lot of these ignores when I get time to work on issue #333.

**Review by pagrubel — APPROVED** [review]

**Pull request merged**